src/liballoc/tests/lib.rs

```diff
@@ -7,6 +7,7 @@
 #![feature(try_reserve)]
 #![feature(unboxed_closures)]
 #![feature(vecdeque_rotate)]
+#![deny(rust_2018_idioms)]
```

Comment:
I don't understand why it's denied here if it was denied in rustbuild already.
Also, why aren't all the `deny(rust_2018_idioms)`s in specific crates removed?

Reply:
We can remove them in a follow up. However, I personally prefer to keep them as it works better when you `cargo check` a crate without `x.py`.